API.md

```diff
@@ -1122,11 +1134,43 @@ Returns:
 ```
 
 ---
-### Delete Respondent Tokens
-* `DELETE /party-api/v1/respondents/<respondent_id>/verification-tokens/<token>`
-  * Updates a respondents valid verification tokens based on `party_uuid` deduced from the `email` passed.
+### Delete Respondent Token
+* `DELETE /party-api/v1/respondents/<respondent_id>/verification-token/<token>`
+  * Updates a respondent's valid verification token based on `party_uuid` deduced from the `email` passed.
+
+---
 
 ---
+### Get Password Reset Counter
+* `GET /party-api/v1/respondents/<respondent_id>/password-reset-counter`
+  * Retrieves a respondent's password reset counter based on the `party_uuid`
+
+### Success
+```json
+{
+  "counter": <counter>
+}
+```
+---
+
+---
+### Put Password Reset Counter
+* `PUT /party-api/v1/respondents/<respondent_id>/password-reset-counter`
```

Comment:
Why do you need this?  If a user attempts to get a password reset email then it should just increment itself without something external telling it to increment.  Or am I missing something?